libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/preview/CarRouteLine.kt

```diff
@@ -114,12 +121,25 @@ class CarRouteLine internal constructor(
             locationPlugin.addOnIndicatorPositionChangedListener(onPositionChangedListener)
             mainCarContext.mapboxNavigation.registerRouteProgressObserver(routeProgressObserver)
             routesProvider.registerRoutesListener(routesListener)
+
+            styleScope.coroutineContext.cancelChildren()
```

Comment:
```suggestion
```

This should be unnecessary. The `MapboxCarMap` should never call `onAttached` more than once. If it does, we should fix the `MapboxCarMap`

Reply:
This is actually for the case when style changes while the map is attached.

Reply:
oh right, can see the inner block 👍